### PR TITLE
The `preprocess` script fix

### PR DESCRIPTION
**Pull request by cy6eria**
This PR belongs to my #4863 issue.

### CODE REVIEW

**Comment by archmoj** [comment]
In regard with https://github.com/plotly/plotly.js/issues/4863#issuecomment-633587743, right now we use node 12, not 14. So I am not sure if we could move forward with this PR at this moment.
cc: @alexcjohnson

**Comment by alexcjohnson** [comment]
Thanks for the investigation and PR @cy6eria!

@archmoj The tests pass in this PR, do they exercise `preprocess` / `update_version`? If this patch works on both node 12 and node 14, it seems like a good idea to accept it, as it will make things easier once we're ready to bump to 14. If it breaks our workflow on 12 then we can leave this PR open and use it as the base for our eventual upgrade to 14.

File: tasks/util/update_version.js [hunk 1]
```diff
@@ -11,7 +11,7 @@ module.exports = function updateVersion(pathToFile) {
             if(isVersionNode(node)) node.update('\'' + pkg.version + '\'');
         });
 
-        fs.writeFile(pathToFile, out, function(err) {
+        fs.writeFile(pathToFile, out.toString(), function(err) {
```
Comment (archmoj) [inline tasks/util/update_version.js:14]: Let's test for the type of `out` to be `object` before casting to string.

Comment (cy6eria) [reply tasks/util/update_version.js:14]: Hey! Sorry for a long answer. I don't think that it's necessary. It doesn't matter what `out` is. It inherit `toString()` method from `Object`. Even if it's string it has a `toString` method.

**Pull request merged**